code-of-conduct-languages/hi.md

```diff
@@ -0,0 +1,30 @@
+## CNCF कम्युनिटी कोड ओफ़ कंडक्ट v1.0
+
+### कंट्रीब्यूटर कोड ओफ़ कंडक्ट
+
+इस प्रॉजेक्ट के कंट्रीब्यूटर और मेंटेनर के रूप में, और एक कम्युनिटी को बढ़ावा देने के हित में, हम उन सभी लोगों का सम्मान करने की प्रतिज्ञा करते हैं जो रिपोर्टिंग issues के माध्यम से योगदान करते हैं, फीचर request पोस्ट करते हैं, प्रलेखन अपडेट करते हैं, pull requset या patches, और अन्य गतिविधियों में भाग लेते हैं।
+
+हम इस प्रॉजेक्ट में सभी के लिए हरैस्मेंट-फ्री अनुभव बनाने के लिए प्रतिबद्ध हैं, भले ही अनुभव के स्तर, जेंडर, जेंडर आइडेंटिटी और एक्सप्रेशन, सेक्सुअल ओरिएंटेशन, डिसेबिलिटी, पर्सनल apperance, शरीर का आकार, नस्ल, अनैतिक, आयु, धर्म, या राष्ट्रीयता।
+
+प्रतिभागियों द्वारा अस्वीकार्य व्यवहार के उदाहरणों में शामिल हैं:
+
+* लैंगिक भाषा या कल्पना का उपयोग
+* व्यक्तिगत हमले
+* ट्रोलिंग or अपमानजनक/derogatory टिप्पणी
+* सार्वजनिक या निजी उत्पीड़न
+* अन्य निजी जानकारी, जैसे कि भौतिक या electronic addresses, बिना स्पष्ट अनुमति के प्रकाशित करना
+* अन्य unethical या अव्यवसायिक आचरण।
+
+प्रोजेक्ट मेंटेनर के पास टिप्पणियों को हटाने, संपादित, या कोड, विकी संपादन, मुद्दे और अन्य योगदान अस्वीकार करने का अधिकार और जिम्मेदारी है, जो इस कोड ओफ़ कंडक्ट में संरेखित नहीं हैं। इस कोड ओफ़ कंडक्ट को अपनाने से, प्रॉजेक्ट के रखवाले इस प्रॉजेक्ट के प्रबंधन के हर पहलू के लिए इन सिद्धांतों को उचित और लगातार लागू करने के लिए खुद को प्रतिबद्ध करते हैं। प्रॉजेक्ट अनुचर जो कोड ओफ़ कंडक्ट का पालन नहीं करते हैं या लागू नहीं करते हैं, उन्हें प्रॉजेक्ट टीम से स्थायी रूप से हटाया जा सकता है।
+
+यह कोड ओफ़ कंडक्ट प्रॉजेक्ट के रिक्त स्थान और सार्वजनिक स्थानों पर लागू होती है जब कोई व्यक्ति प्रॉजेक्ट या उसके समुदाय का प्रतिनिधित्व करता है।
+
+कुबरनेट्स में अपमानजनक, उत्पीड़न या अन्यथा अस्वीकार्य व्यवहार के उदाहरणों को <conduct@kubernetes.io> के माध्यम से [कुबरनेट्स कोड ओफ़ कंडक्ट समिति](https://git.k8s.io/community/committee-code-of-conduct) से संपर्क करके सूचित किया जा सकता है। अन्य प्रॉजेक्टओं के लिए, कृपया किसी सीएनसीएफ प्रोजेक्ट मेंटेनर या हमारे मध्यस्थ, मिशी चौधरी <mishi@linux.com> से संपर्क करें।
```

Comment:
Is this प्रोजेक्टों in ...अन्य प्रॉजेक्टओं के लिए... ?

Reply:
There is a spacing between `प्रॉजेक्ट` & `ओं`.

Reply:
What does that mean, you should do it प्रोजेक्टों,  प्रॉजेक्ट ओं is nothing. ओं is never used in this way, it is always added with a word, it is _maatra_!

Reply:
Ah! I see. Thanks.